anndata/_core/anndata.py

```diff
@@ -1124,7 +1124,8 @@ def _remove_unused_categories(self, df_full, df_sub, uns):
             if not is_categorical_dtype(df_full[k]):
                 continue
             all_categories = df_full[k].cat.categories
-            df_sub[k] = df_sub[k].cat.remove_unused_categories()
+            with pd.option_context("mode.chained_assignment", None):
+                df_sub[k] = df_sub[k].cat.remove_unused_categories()
```

Comment:
why isn't this different from what's on master?

Reply:
Because my local clone got messed up...... Still not sure how, but my local `master` had started tracking my forks `master`